src/cartservice.dockerfile

```diff
@@ -5,20 +5,18 @@ ENV CGO_ENABLED=0 GOOS=linux
 WORKDIR /go/src/hipstershop
 
 # Build Go binary
-COPY Makefile go.mod go.sum ./
+COPY ./cartservice ./cartservice
```

Comment:
all the changes here and in the other services are for the `libraries` folder creation and moving packages inside of it.